packages/rancher-tracing/package.yaml

```diff
@@ -1,3 +1,3 @@
 url: local
-packageVersion: 02
+packageVersion: 03
```

Comment:
@brendarearden should we be bumping the packageVersion here or should we just modify the Chart.yaml version to release a new version and reset these to 00 / 00?

@cbron this might also be a question for you: for local charts, what should our convention be w.r.t. bumping packageVersion v.s. bumping the major.minor.patch version? For pushprox, I'm leaning towards never touching the packageVersion and only touching the major.minor.patch + releaseCandidate version, re: [this comment](https://github.com/rancher/charts/pull/998#discussion_r575613219).